cmd/otelcol/config/collector/config.d.linux/receivers/redis.discovery.yaml

```diff
@@ -38,7 +38,6 @@
 #         log_record:
 #           body: >-
 #             Make sure your user credentials are correctly specified using the
-#             `SPLUNK_DISCOVERY_RECEIVERS_redis_CONFIG_password="<password>"` and
```

Comment:
Should we be adding the username config option here to replace the duplicate?

Reply:
There is no `username` config option for redis receiver. It needs password only